src/views/AppView.vala

```diff
@@ -49,7 +49,7 @@ namespace Cipher.Views {
             var hash = new HashView ();
 
             var main_stack = new Gtk.Stack ();
-            main_stack.add_titled (welcome, "welcome", "Welcome");
+            main_stack.add_titled (welcome, _("welcome"), _("Welcome"));
```

Comment:
```suggestion
            main_stack.add_titled (welcome, "welcome", _("Welcome"));
```

The second argument is a name for child which used in codes, so it should not be translatable. See https://valadoc.org/gtk+-3.0/Gtk.Stack.add_titled.html